docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -41,43 +32,20 @@ Within every game (XML) file, inside the "games" folder inside the main folder o
 - `="Battleships repair at beginning of round"` >> `="Units Repair Hits Start Turn"`
 - `="Naval Bombard Casualties Return Fire Restricted"` >> `="Naval Bombard Casualties Return Fire"`
 - `="SBR Affects Unit Production"` >> `="Damage From Bombing Done To Units Instead Of Territories"`
-
-### Changes of game (XML) file special code for the above items:
-
-_(these are cases of advanced coding, so you probably don't have these within the game file)_
-
-Within every game file (XML), inside the "games" folder inside the main folder of the map, change:
-
 - `="conditionType" count="XOR"` >> `="conditionType" count="1"`
 - `="conditionType" count="-reset-XOR"` >> `="conditionType" count="-reset-1"`
-
-### Changes of already deprecated game (XML) file items:
-
-_(these items were already deprecated, so you shouldn't have had these within the game file already)_
-
-Within every game (XML) file, inside the "games" folder inside the main folder of the map, change:
-
-- `="isTwoHit" value="true"` >> `="hitPoints" value="2"`
-- `="isTwoHit" value="false"` >> `="hitPoints" value="1"`
-- `="occupiedTerrOf"` >> `="originalOwner"`
-- `="victoryCity" value="true"` >> `="victoryCity" value="1"`
-- `="victoryCity" value="false"` >> `="victoryCity" value="0"`
-
-### Changes of game (XML) file special code for the above items:
-
-_(these are cases of advanced coding, so you probably don't have these within the game file)_
-
-Within every game file (XML), inside the "games" folder inside the main folder of the map, change:
-
 - `="isTwoHit" count="true"` >> `="hitPoints" count="2"`
+- `="isTwoHit" value="false"` >> `="hitPoints" value="1"`
 - `="isTwoHit" count="-reset-true"` >> `="hitPoints" count="-reset-2"`
 - `="isTwoHit" count="false"` >> `="hitPoints" count="1"`
 - `="isTwoHit" count="-reset-false"` >> `="hitPoints" count="-reset-1"`
+- `="occupiedTerrOf"` >> `="originalOwner"`
 - `="victoryCity" count="true"` >> `="victoryCity" count="1"`
 - `="victoryCity" count="-reset-true"` >> `="victoryCity" count="-reset-1"`
 - `="victoryCity" count="false"` >> `="victoryCity" count="0"`
 - `="victoryCity" count="-reset-false"` >> `="victoryCity" count="-reset-0"`
 
+
 ### XML Deletions
 
 _(these items were already deprecated and did nothing)_
```

Comment:
You previously removed a "were already deprecated" reference. I suggest either removing all or none of them. Also saying that "did nothing" is meaningless in the moment it is not documented when (this was supposed to mean "did nothing in 1.8.0.9", that being the referenced starting point).

Reply:
That context was not clear, this section perhaps should then instead give what these attributes should have been. I see now there was an assumption that maps being upgraded to 1.8.0.9 were done in a sloppy manner and instead of replacing legacy attributes only new attributes were added. That is a slightly confusing assumption I think, hence if we just give what the values should be, we can leave it to map makers to de-dupe the options (but duplicate options won't hurt the engine regardless so it's more important then we give the correct upgrade)

Reply:
Well, at least in the moment that now this document is not specifying any particular version, I believe it is obvious that saying "were already deprecated and did nothing" is meaningless, as it is really referring to nothing. Assuming they are currently deprecated and do nothing, there is most likely some version of TripleA in the past where they weren't. A line like this has any sense only as long as the document itself is referencing a TripleA version from which to upgrade (where these items would have been already deprecated and useless). Strictly speaking, one may argue that, in the original document, they were no longer deprecated in the final TripleA version, since they went from being deprecated to being completely removed (something that I understant now has been effectively reverted, going back to the 1.8.0.9 situation).